Resources/services.xml

```diff
@@ -3,11 +3,15 @@
 <container xmlns="http://symfony.com/schema/dic/services"
            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <parameters>
+        <parameter key="shyim_web_p.cached_download_dir">%shyim_web_p.plugin_dir%/.libwebp</parameter>
```

Comment:
I don't like the idea to save the binary in the Plugin Folder.
- The Plugin Folder should be readOnly
- A upgrade of the Plugin would delete the binary

I would suggest to move it instead in the bin Folder where also the console command is.

Reply:
A plugin directory is cleared on an update? I though one has to remove files on update that got removed in vcs.

I'd rather put it in the var-folder instead because it contains a whole package with other files than just executables. So the bin folder is not filled with non-executables. Any hints to get that path? `Shopware()->DocPath('var')`?

Reply:
%kernel.root_dir%/var/webp

Reply:
Since when does `%kernel.root_dir%` exist?

Reply:
I guess shopware 4

Reply:
That suits